app/src/main/AndroidManifest.xml

```diff
@@ -16,17 +16,14 @@
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
         <activity
-            android:name="Activities.Login"
+            android:name="Activities.PatientHomeMenuActivity"
```

Comment:
Could we leave Login as the main activity in the manifest?

Reply:
yes, sorry I was testing the activity to see if they work